database/mssql/scripts/dbscripts/PSP_PIMS_S17_00/Alter Up/01_PSP_PIMS_S16_01_Alter_Up.sql

```diff
@@ -0,0 +1,940 @@
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop index dbo.INSRNC_INSURER_CONTACT_ID_IDX
+PRINT N'Drop index dbo.INSRNC_INSURER_CONTACT_ID_IDX'
+GO
+DROP INDEX [dbo].[PIMS_INSURANCE].[INSRNC_INSURER_CONTACT_ID_IDX]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop index dbo.INSRNC_INSURANCE_PAYEE_TYPE_CODE_IDX
+PRINT N'Drop index dbo.INSRNC_INSURANCE_PAYEE_TYPE_CODE_IDX'
+GO
+DROP INDEX [dbo].[PIMS_INSURANCE].[INSRNC_INSURANCE_PAYEE_TYPE_CODE_IDX]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop index dbo.INSRNC_BCTFA_RISK_MGMT_CONTACT_ID_IDX
+PRINT N'Drop index dbo.INSRNC_BCTFA_RISK_MGMT_CONTACT_ID_IDX'
+GO
+DROP INDEX [dbo].[PIMS_INSURANCE].[INSRNC_BCTFA_RISK_MGMT_CONTACT_ID_IDX]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop trigger dbo.PIMS_INSRNC_I_S_I_TR
+PRINT N'Drop trigger dbo.PIMS_INSRNC_I_S_I_TR'
+GO
+DROP TRIGGER [dbo].[PIMS_INSRNC_I_S_I_TR]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop trigger dbo.PIMS_INSPAY_I_S_U_TR
+PRINT N'Drop trigger dbo.PIMS_INSPAY_I_S_U_TR'
+GO
+DROP TRIGGER [dbo].[PIMS_INSPAY_I_S_U_TR]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop trigger dbo.PIMS_INSPAY_I_S_I_TR
+PRINT N'Drop trigger dbo.PIMS_INSPAY_I_S_I_TR'
+GO
+DROP TRIGGER [dbo].[PIMS_INSPAY_I_S_I_TR]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop trigger dbo.PIMS_PERSON_A_S_IUD_TR
+PRINT N'Drop trigger dbo.PIMS_PERSON_A_S_IUD_TR'
+GO
+DROP TRIGGER [dbo].[PIMS_PERSON_A_S_IUD_TR]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop trigger dbo.PIMS_PIMPRV_A_S_IUD_TR
+PRINT N'Drop trigger dbo.PIMS_PIMPRV_A_S_IUD_TR'
+GO
+DROP TRIGGER [dbo].[PIMS_PIMPRV_A_S_IUD_TR]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop view dbo.PIMS_PROPERTY_BOUNDARY_VW
+PRINT N'Drop view dbo.PIMS_PROPERTY_BOUNDARY_VW'
+GO
+DROP VIEW [dbo].[PIMS_PROPERTY_BOUNDARY_VW]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop view dbo.PIMS_CONTACT_MGR_VW
+PRINT N'Drop view dbo.PIMS_CONTACT_MGR_VW'
+GO
+DROP VIEW [dbo].[PIMS_CONTACT_MGR_VW]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.INSPAY_PK
+PRINT N'Drop primary key constraint dbo.INSPAY_PK'
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE_PAYEE_TYPE]
+	DROP CONSTRAINT [INSPAY_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_LEASE
+PRINT N'Alter table dbo.PIMS_LEASE'
+GO
+ALTER TABLE [dbo].[PIMS_LEASE] ALTER COLUMN [LEASE_CATEGORY_TYPE_CODE] nvarchar(20) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_LEASE] ALTER COLUMN [LEASE_RESPONSIBILITY_TYPE_CODE] nvarchar(20) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_LEASE] ALTER COLUMN [LEASE_PMT_FREQ_TYPE_CODE] nvarchar(20) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_PROPERTY_IMPROVEMENT
+PRINT N'Alter table dbo.PIMS_PROPERTY_IMPROVEMENT'
+GO
+ALTER TABLE [dbo].[PIMS_PROPERTY_IMPROVEMENT] ALTER COLUMN [IMPROVEMENT_DESCRIPTION] nvarchar(2000) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+EXEC dbo.sp_rename N'dbo.PIMS_PROPERTY_IMPROVEMENT.UNIT' , N'ADDRESS', 'COLUMN'
```

Comment:
@asanchezr is the unit field used by the address page?

Reply:
Yes it is @devinleighsmith 
This DB change will require fixing backend mappings and frontend code as well. Do we know the rationale behind this specific rename @dfilteau ?

Reply:
As with all my changes, they're at the request of the business.  For further information please contact Kartik.